### ISSUE TITLE
Report using `error` without key

### ISSUE THREAD

**Issue by maratori** [go-simpler/sloglint#39]
`slog` doesn't catch the following common mistake 
```go
err := errors.New("failed")
// wrong
logger.Error("Something went wrong", err) // "!BADKEY":"failed"
// correct
logger.Error("Something went wrong", "error", err) // "error":"failed"
```

`err` may be any type except `slog.Attr`. So ideally, need to validate any type. But the most common is `error`, and need to validate at least it.

**Comment by tmzane** [comment]
Hello,

Looks like it is already reported at https://github.com/golang/go/issues/65740 and about to get fixed.
